src/Makefile

```diff
@@ -34,8 +34,8 @@ DATE_STRING := $(shell date +%Y.%m.%d.%H:%M:%S)
 # same path of THIS Makefile
 SRCDIR := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
 
-include $(SRCDIR)/config.mk
 include $(SRCDIR)/dpdk.mk
+include $(SRCDIR)/config.mk
```

Comment:
what is the purpose to  move include $(SRCDIR)/config.mk after  $(SRCDIR)/dpdk.mk?
it may cause the problem that CONFIG_MLX5 not work after changing the order.

Reply:
Well, the `$(RTE_SDK)/mk/rte.vars.mk` included in `dpdk.mk` overrides `CFLAGS`, so we need to include `dpdk.mk` first. Maybe `CONFIG_MLX5` should be handled in another way that works well with the new include ordering.

Reply:
编译说明的文档不打算提一下 CONFIG_MLX5 吗？我用 1.8.8，MLX5_CORE PMD，按照入门文档，搞了 3 天才发现 原来还有 CONFIG_MLX5 这个配置项卡了一道